src/dbc.rs

```diff
@@ -66,9 +66,15 @@ use crate::{
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub struct Dbc {
+    /// Encrypted information for and about this DBC's owner
     pub content: DbcContent,
+    /// The public key (owner) of this DBC, used for SpentProofs
+    pub public_key: PublicKey,
```

Comment:
Is this not accessible through the `DbcContent`?

Reply:
Only accessible by the DBC owner through the encrypted derivation index (as it used to be a secret). 
This whole thing should be reworked and maybe we should drop that part altogether later. 
But for now, making it public is important for usability.